src/arrayMethodSort.js

```diff
@@ -4,8 +4,22 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const defaultCompare = function defaultCompare(a, b) {
+    return a.toString() > b.toString();
+  };
```

Comment:
The default comparison function should return a number, not a boolean. It should return a negative value if the first argument is less than the second, zero if they're equal, and a positive value if the first is greater.